Cargo.toml

```diff
@@ -15,4 +15,4 @@ keywords = ["data-structures"]
 readme = "README.md"
 
 [dependencies]
-linked-hash-map = "*"
+linked-hash-map = { git = "https://github.com/contain-rs/linked-hash-map" }
```

Comment:
And you also need to publish a new version of `lru-cache`, because of the deprecation of `HashState`.